dotnet/samples/KernelSyntaxExamples/Example01_MethodFunctions.cs

```diff
@@ -1,22 +1,30 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
-using System;
 using System.Threading.Tasks;
 using Microsoft.SemanticKernel.Plugins.Core;
+using Xunit;
+using Xunit.Abstractions;
 
-public static class Example01_MethodFunctions
+namespace Examples;
+
+public class Example01_MethodFunctions : BaseTest
 {
-    public static Task RunAsync()
+    public Example01_MethodFunctions(ITestOutputHelper output) : base(output)
+    {
+    }
```

Comment:
If we cannot avoid these boilerplate constructors, we can consider prettifying them using C# 12 primary constructor syntax: `public class Example01_MethodFunctions(ITestOutputHelper output) : BaseTest(output)`.